src/utilities/util_verify.c

```diff
@@ -157,7 +157,8 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
              * entries to prevent unnecessary work. Skip over the double up entries and also any
              * entries that are not supported with verify.
              */
-            if (WT_PREFIX_MATCH(key, "table:") && !WT_PREFIX_MATCH(key, WT_SYSTEM_PREFIX)) {
+            if ((WT_PREFIX_MATCH(key, "table:") || WT_PREFIX_MATCH(key, "layered:")) &&
+              !WT_PREFIX_MATCH(key, WT_SYSTEM_PREFIX)) {
```

Comment:
Wouldn't `WT_PREFIX_MATCH(key, WT_SYSTEM_PREFIX)` be always false if the prefix matches anything in the first subexpression?

Reply:
i removed it 1d308279816719976f67a00d37bf0fc85cee7e40